src/deposit.cairo

```diff
@@ -195,24 +251,24 @@ mod Deposit {
                 contract_address: reserve_data.z_token_address
             };
             let contract_address = get_contract_address();
-            let mut debt = zk_market.get_user_debt_for_token(contract_address, debt_token);
+            let mut debt = zk_market.get_user_debt_for_token(contract_address, debt_token).into();
 
             let debt_dispatcher = ERC20ABIDispatcher { contract_address: debt_token };
             let (supply_decimals, debt_decimals) = (
                 fast_power(10, token_disp.decimals().into()),
                 fast_power(10, debt_dispatcher.decimals().into())
             );
-            let (SQRT_LIMIT_REPAY, SQRT_LIMIT_WITHDRAW) = if supply_token == pool_key.token0 {
-                (18446748437148339061, 6277100250585753475930931601400621808602321654880405518632)
+            let SQRT_LIMIT_REPAY = if supply_token == pool_key.token0 {
+                18446748437148339061
             } else {
-                (6277100250585753475930931601400621808602321654880405518632, 18446748437148339061)
+                6277100250585753475930931601400621808602321654880405518632
```

Comment:
This is essentially the max slippage no? Is it correct to hardcode it?

Reply:
I think I will calculate 3-5% slippage from current sqrt_ratio